endpoint.bal

```diff
@@ -90,19 +94,22 @@ public client class Client {
     # + queueResourcePath - Resource path to the queue from the host address. e.g.: /610968236798/myQueue.fifo
     # + attributes - Non-mandatory parameters for sending a message 
     # + return - If success, details of the sent message, else returns error
-    public remote function sendMessage(string messageBody, string queueResourcePath, map<string> attributes) 
-            returns @tainted OutboundMessage|OperationError {
-        string|error msgbody = encoding:encodeUriComponent(messageBody, UTF_8);
+    @display {label: "Send message in queue"}
+    remote isolated function sendMessage(@display {label: "Message body to send"} string messageBody, 
+                                @display {label: "Resource path to queue"} string queueResourcePath, 
+                                @display {label: "Map of attributes"} map<string> attributes) 
```

Comment:
Make attributes as optional. Are they mandatory?

Reply:
Shall we model them as optional parameters? Then it is more convenient and user friendly from language perspective as well as UI perspective. 
Apply this for other operations as well where necessary.

Reply:
Please check my above comment. You have to follow up the API Reference doc and implement all parameters in the sendMessage operation.

Reply:
Made those optional